tools/wagmi-example/README.md

```diff
@@ -0,0 +1,21 @@
+# Wagmi Hedera Example
```

Comment:
nit: should add a brief summary on what Wagmi is an add a link.
For some developers this may be their first exposure

Reply:
@Nana-EC will this summary be enough? https://github.com/hashgraph/hedera-json-rpc-relay/pull/3000/files#diff-bd7c3891dc764da67cba302d66c3bb940bc8d0851095de79357802669a2c2727R3 I've included the link as well.